examples/webgl_animation_skinning_blending.html

```diff
@@ -23,6 +23,7 @@
 				text-decoration: line-through;
 			}
 		</style>
+		<link rel="preload" href="models/gltf/Soldier.glb" as="fetch" crossorigin>
```

Comment:
Perhaps including `type="application/octet-stream"` would fix Safari?

Reply:
^That's not a universal answer: the official MIME type is actually `model/gltf-binary`. But not many servers know about model MIME types, so I think we need to use what this server actually supports.

Reply:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content#Including_a_MIME_type):

> the browser will use the type attribute value to work out if it supports that resource, and will only download it if so, ignoring it if not.

So it seems like the type is used for linking to multiple resource types, e.g. .mp4 and .webm. That means it _shouldn't_ be required here. Can't hurt to test though in can Safari has different behavior. 

As for server support, this implies that it's used in the browser to decide whether to request the file, so it shouldn't matter whether the mime-type is configured on the server.

Reply:
By the way, is `crossorigin` needed here? I thought that was only required for fonts.

Reply:
> By the way, is crossorigin needed here?

It's needed only for local testing with `npm run dev`.

Reply:
crossorigin must match the way it is requested later. if you preload without crossorgin and then load with crossorigin, you get the complaint that there was a mis-match, and preloaded response is not used. I did run into it elsewhere, so I put crossorigin here without thinking.

Reply:
It is possible that we faced with bug in Safari? Should we open an issue in WebKit?
https://bugs.webkit.org/
Because the same code works without double loading in Chrome and Firefox.

Reply:
Exactly the same error appear in chrome with wrong `as="image/fetch"` and the same error we see in Safari with current attributes in link. Preload works but in a wrong way.